cpp/vcpkg.json

```diff
@@ -156,7 +156,7 @@
     { "name": "boost-vcpkg-helpers", "version": "1.80.0#1" },
     { "name": "boost-winapi", "version": "1.80.0#1" },
     { "name": "civetweb", "version": "1.15#2" },
-    { "name": "curl", "version": "7.86.0#1" },
+    { "name": "curl", "version": "7.88.1#1" },
```

Comment:
This should now be moved above the comment on line 78

Reply:
I'm just going to remove the comments, I don't think they add much value and it's a maintenance burden to keep them correct.